infrastructures/openshift/src/test/java/org/eclipse/che/workspace/infrastructure/openshift/multiuser/oauth/IdentityProviderConfigBuilderTest.java

```diff
@@ -225,7 +225,8 @@ public void testRethrowOnInvalidTokenBadRequestException() throws Exception {
 
   @Test(expectedExceptions = {InfrastructureException.class})
   public void testRethrowOnAnyException() throws Exception {
-    when(keycloakServiceClient.getIdentityProviderToken(anyString())).thenThrow(Exception.class);
```

Comment:
Can you elaborate on why we need this change? I don't see any connection with the purpose of the PR.